src/core/messenger/resize.ts

```diff
@@ -67,6 +67,7 @@ const resize = (
 
 		if (iframeContainer) {
 			Object.assign(iframeContainer.style, styles);
+			adSlot.style.maxHeight = 'none';
```

Comment:
Should this be in the `iframeContainer` conditional?

Reply:
I thought this would be appropriate, as we need an iFrameContainer to be present in order to apply the styling?
Do you mean to add the styling regardless of conditions?
```js
return fastdom.mutate(() => {
    Object.assign(iframe.style, styles);

    if (iframeContainer) {
        Object.assign(iframeContainer.style, styles);
    }
    
    adSlot.style.maxHeight = 'none'; // Applied regardless of iframeContainer
});
```